app/views/rubygems/show.html.erb

```diff
@@ -43,8 +43,8 @@
         </div>
       <% end %>
 
-      <%= render :partial => "rubygems/dependencies", :locals => { :dependencies => @latest_version.dependencies.runtime } %>
```

Comment:
Forgot to mention I went all post-1.9 on these hashes. It seemed unnecessary to stick to the old syntax here when ERB lines tend to be too long for comfort.